webapi/webapi-nativefilesystem-xwalk-tests/tests.full.xml

```diff
@@ -627,6 +627,18 @@
           </spec>
         </specs>
       </testcase>
+      <testcase component="Crosswalk APIs/Experimental/Native File System" execution_type="auto" id="NativeFileSystem_getRealPath" priority="P3" purpose="Check if getRealPath function returns the full path of this virtual root" status="approved" type="compliance" subcase="4">
+        <description>
+          <test_script_entry>/opt/webapi-nativefilesystem-xwalk-tests/nativefilesystem/NativeFileSystem_getRealPath.html</test_script_entry>
+        </description>
+        <specs>
+          <spec>
+            <spec_assertion category="Tizen W3C API Specifications" element_name="getRealPath" element_type="method" interface="NativeFileSystem" section="Storage" specification="Native File System" />
```

Comment:
This spec isn't `Tizen W3C API Specifications`. Anyway, let's update it in a new PR.